clouditor-engine-core/src/main/java/io/clouditor/assurance/EvaluationResult.java

```diff
@@ -30,20 +30,60 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.clouditor.assurance.ccl.Condition;
+import io.clouditor.data_access_layer.PersistentObject;
 import io.clouditor.discovery.AssetProperties;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
+import javax.persistence.*;
 import javax.validation.constraints.NotNull;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
 
-public class EvaluationResult {
+@Entity(name = "evaluation_result")
+@Table(name = "evaluation_result")
+public class EvaluationResult implements PersistentObject<String> {
+
+  private static final long serialVersionUID = 7255742076812915308L;
+
+  @Id
+  @Column(name = "time_stamp", nullable = false)
+  private final String timeStamp = new Date().toString();
```

Comment:
any particular reason not to use a real date or `Instant` object?

Reply:
Changed to Date